src/util.ts

```diff
@@ -11,6 +11,8 @@ export function convertIcon(iconText: string): string {
   switch (icon) {
     case 'Ko':
       return ':cow2:';
+    case 'Kalv':
+      return ':cow2:';
```

Comment:
Hmmmm, beef ≠ veal? 🤔

Reply:
True, but there's not really a good calf emoji.
I'm open for suggestions 🤔

Reply:
Don't we also need some [extra gifs](https://giphy.com/search/calf) then?

![Calf](https://media.giphy.com/media/LilPRDHg9Qre0/giphy.gif)

Reply:
Updated again